# AircraftState.msg

Header header
float64 v_ias
geometry_msgs/Vector3 euler
geometry_msgs/Vector3 omega
geometry_msgs/Vector3 v_gps
geometry_msgs/Vector3 body_accel
geometry_msgs/Vector3 ac_lla
bool is_lost_comm

float64 throttle
float64 aileron
float64 elevator
float64 rudder
float64 flap
